certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificate/model/Certificate.java

```diff
@@ -37,6 +38,7 @@ public class Certificate {
   private Xml xml;
   private Sent sent;
   private Revoked revoked;
+  private ExternalReference externalReference;
```

Comment:
ExternalReference is both in Certificate and CertificateMetadata. I would like to see it as part of Certificate only.